desktop/src/main/java/bisq/desktop/main/market/offerbook/OfferBookChartViewModel.java

```diff
@@ -355,6 +333,26 @@ private void updateChartData() {
         buildChartAndTableEntries(allSellOffers, OfferPayload.Direction.SELL, sellData, topSellOfferList);
     }
 
+    private Comparator<Offer> getComparator(boolean reversePrimarySortOrder) {
+        Comparator<Offer> primary = Comparator.comparing(Offer::getPrice, (o1, o2) -> {
+            long a = o1 != null ? o1.getValue() : 0;
+            long b = o2 != null ? o2.getValue() : 0;
+            if (a != b) {
+                if (o1 != null && CurrencyUtil.isCryptoCurrency(o1.getCurrencyCode()))
+                    return a > b ? 1 : -1;
+                else
+                    return a < b ? 1 : -1;
+            } else {
+                return 0;
+            }
+        });
+
+        if (reversePrimarySortOrder)
+            primary = primary.reversed();
+
+        return primary.thenComparing(Offer::getAmount);
```

Comment:
Minor suggestion to ensure orders with higher amounts are displayed first:

```suggestion
        // 1st comparator: sort by price
        // - ascending, if showing orders to sellers
        // - descending, if showing orders to buyers
        if (reversePrimarySortOrder)
            primary = primary.reversed();

        // 2nd comparator: sort by amount, in descending order
        // The goal is to show the more attractive offers at the top
        // Both buyers and sellers would prefer to see higher amounts first (for orders with the same price)
        Comparator<Offer> secondary = Comparator.comparing(Offer::getAmount, Comparator.reverseOrder());

        return primary.thenComparing(secondary);
```

Reply:
I think this is the more natural way to show the order book, larger orders first. That way more of the book is shown sooner.